cypress/e2e/smoke/address_book.cy.js

```diff
@@ -30,8 +29,7 @@ describe('Address book', () => {
       // Add a new entry manually
       cy.contains('Create entry').click()
       cy.get('input[name="name"]').type(NAME)
-      cy.get('input[name="address"]').type(ENS_NAME)
-      // cy.wait(5000)
+      cy.get('input[name="address"]').type(`gor:${ENS_ADDRESS}`)
```

Comment:
This seems to be a bug in the form. I was investigating on it but I will fill in an issue